charts/camunda-platform-alpha/test/integration/testsuites/vars/files/testsuite-core.yaml

```diff
@@ -305,3 +311,62 @@ testcases:
 #     - result.body ShouldContainSubstring optimize
 #     - result.body ShouldContainSubstring web-modeler-restapi
 #     - result.body ShouldContainSubstring core
+#     - result.body ShouldContainSubstring zeebe
+#     - result.body ShouldContainSubstring zeebe-gateway
+#############################################################################
+#- name: run command svc/camunda-zeebe-gateway
+#  steps:
+#  - type: exec
+#    script: "kubectl port-forward svc/camunda-zeebe-gateway 8080:8080 > /dev/null 2>&1 &  curl localhost:8080"
+#- name: Check camunda-operate
+#  steps:
+#  - type: http
+#    method: GET
+#    url: "http://localhost:8081"
+#    retry: 2
+#    delay: 1
+  ########################################################Port forwarding Camunda components##################################
+- name: Port forwarding Camunda components
+  steps:
+  - name: "Port forward to {{ .value.component }} service "
+    type: exec
+    range:
+    - component: zeebe-gateway
```

Comment:
Write the component name like this `Zeebe Gateway` (the same for all components, capitalized).